gcc/config/riscv/vector.md

```diff
@@ -274,6 +274,16 @@
   [(set_attr "type" "vector")
    (set_attr "mode" "none")])
 
+(define_insn "vsetvlmax<VIMODES:mode>_<P:mode>"
+  [(set (reg:VIMODES VTYPE_REGNUM) (const_int UNSPECV_VSETVL))
+   (set (reg:P VL_REGNUM) (const_int UNSPECV_VSETVL))
+   (set (match_operand:P 0 "register_operand" "=r")
+	(reg:P VL_REGNUM))]
+  "TARGET_VECTOR"
+  "vsetvli\t%0,x0,e<VIMODES:sew>,m<VIMODES:lmul>"
+  [(set_attr "type" "vset")
```

Comment:
`type` should be `vector`, I guess that `type` used for your internal branch :p

Reply:
Indeed :)

Fix it